go-controller/pkg/ovn/address_set_test.go

```diff
@@ -24,6 +24,10 @@ func (asn *testAddressSetName) makeName() string {
 	return fmt.Sprintf("%s.%s.%s", asn.namespace, asn.suffix1, asn.suffix2)
 }
 
+const (
+	fakeUUIDv6 = "8a86f6d8-7972-4253-b0bd-ddbef66e9304"
```

Comment:
should this constant be defined in ovn_test.go instead?

Reply:
Done.